frontend/src/app/modules/home/dashboard/tile-menu/tile-menu.component.html

```diff
@@ -11,34 +11,26 @@
     <!--    Center Buttons    -->
     <div class="center-buttons">
         <!--    Text Button    -->
-        <button (click)="textMenu.toggle($event)" [disabled]="!menuTextItems.length"
-                class="dropdown-button p-button-outlined" icon="pi pi-bars" pButton pTooltip="Text"
+        <button (click)="createHeatMap()"
+                class="dropdown-button p-button-outlined" icon="pi pi-heart" pButton pTooltip="HeatMap"
                 tooltipPosition="bottom" tooltipStyleClass="tooltip-text" type="button">
         </button>
-        <p-tieredMenu #textMenu [model]="menuTextItems" [popup]="true" appendTo="body" styleClass="tiered-menu">
-        </p-tieredMenu>
         <!--    List Button    -->
-        <button (click)="listMenu.toggle($event)" [disabled]="!menuListItems.length"
+        <button (click)="createActiveIssues()"
                 class="dropdown-button p-button-outlined" icon="pi pi-list"
-                pButton pTooltip="List" tooltipPosition="bottom" tooltipStyleClass="tooltip-text" type="button">
+                pButton pTooltip="Active Issues" tooltipPosition="bottom" tooltipStyleClass="tooltip-text" type="button">
         </button>
-        <p-tieredMenu #listMenu [model]="menuListItems" [popup]="true" appendTo="body" styleClass="tiered-menu">
-        </p-tieredMenu>
         <!--    Number Button    -->
         <div class="number-button">
-            <button (click)="valueMenu.toggle($event)" [disabled]="!menuValueItems.length" class="p-button-outlined"
-                    label="1.2" pButton pTooltip="Value" tooltipPosition="bottom" tooltipStyleClass="tooltip-text"
+            <button (click)="createIssueCount()" class="p-button-outlined" icon="pi pi-th-large"
+                        pButton pTooltip="Issue count" tooltipPosition="bottom" tooltipStyleClass="tooltip-text"
```

Comment:
Please write each attribute for a new line

Reply:
OK